commands/user.go

```diff
@@ -87,6 +87,14 @@ var UpdateUserEmailCmd = &cobra.Command{
 	RunE:    withClient(updateUserEmailCmdF),
 }
 
+var UpdateUserNameCmd = &cobra.Command{
+	Use:     "name [user] [new name]",
+	Short:   "Change name of the user",
+	Long:    "Change name of the user.",
+	Example: "  user name testuser newname",
+	RunE:    withClient(updateUserNameCmdF),
+}
```

Comment:
I'd suggest to change the references to `name` for `username`, to avoid confusions. So this would be something like:

```suggestion
var UpdateUsernameCmd = &cobra.Command{
	Use:     "username [user] [new username]",
	Short:   "Change username of the user",
	Long:    "Change username of the user.",
	Example: "  user username testuser newusername",
	RunE:    withClient(updateUsernameCmdF),
}
```